environments/staging/staging.yml

```diff
@@ -8,4 +8,16 @@ txt: []
 
 srv: []
 
-cname: []
+cname:
```

Comment:
aat uses the aat file not staging, do you want to use this file? i.e. we don't have a staging.platform.hmcts.net cert yet